lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -83,9 +111,21 @@ dampening_suppress_time:
 
 default_information:
   kind: boolean
-  config_get_token_append: '/^default-information originate$/'
-  config_set_append: '<state> default-information originate'
-  default_value: false
+  cli_nexus:
+    config_get_token_append: '/^default-information originate$/'
+    config_set_append: '<state> default-information originate'
+    default_value: true
+  cli_ios_xr:
+    config_set: 'router bgp <asnum>'
+    config_set_append: 
+      - ' vrf <vrf>'
```

Comment:
Here as above, can you inherit from the template and do any XR exclusions in the class?